src/ScreenshotManager.vala

```
@@ -329,6 +351,10 @@ public class Gala.ScreenshotManager : Object {
     }
 
     public async void conceal_text () throws DBusError, IOError {
+        if (!is_redacted_font_available) {
+            return;
```

I think we should probably throw an error here?